error: too few arguments to function 'prussdrv_pru_clear_event'

When **making** the `example_apps` i get the following:

```
make CROSS_COMPILE= 
mkdir -p bin
for dir in  PRU_memAccess_DDR_PRUsharedRAM PRU_memAccessPRUDataRam PRU_PRUtoPRU_Interrupt; do make -C $dir CROSS_COMPILE="" LIBDIR_APP_LOADER="../../app_e
make[1]: Entering directory `/root/pru/am335x_pru_package/pru_sw/example_apps/PRU_memAccess_DDR_PRUsharedRAM'
gcc -Wall -I../../app_loader/include -D__DEBUG -O2 -mtune=cortex-a8 -march=armv7-a -c -o obj/PRU_memAccess_DDR_PRUsharedRAM.o PRU_memAccess_DDR_PRUshared 
PRU_memAccess_DDR_PRUsharedRAM.c: In function 'main':
PRU_memAccess_DDR_PRUsharedRAM.c:162:5: error: too few arguments to function 'prussdrv_pru_clear_event'
../../app_loader/include/prussdrv.h:182:9: note: declared here
make[1]: *** [obj/PRU_memAccess_DDR_PRUsharedRAM.o] Error 1
make[1]: Leaving directory `/root/pru/am335x_pru_package/pru_sw/example_apps/PRU_memAccess_DDR_PRUsharedRAM'
make[1]: Entering directory `/root/pru/am335x_pru_package/pru_sw/example_apps/PRU_memAccessPRUDataRam'
gcc -Wall -I../../app_loader/include -D__DEBUG -O2 -mtune=cortex-a8 -march=armv7-a -c -o obj/PRU_memAccessPRUDataRam.o PRU_memAccessPRUDataRam.c 
PRU_memAccessPRUDataRam.c: In function 'main':
PRU_memAccessPRUDataRam.c:148:5: error: too few arguments to function 'prussdrv_pru_clear_event'
../../app_loader/include/prussdrv.h:182:9: note: declared here
make[1]: *** [obj/PRU_memAccessPRUDataRam.o] Error 1
make[1]: Leaving directory `/root/pru/am335x_pru_package/pru_sw/example_apps/PRU_memAccessPRUDataRam'
make[1]: Entering directory `/root/pru/am335x_pru_package/pru_sw/example_apps/PRU_PRUtoPRU_Interrupt'
gcc -I../../app_loader/include -D__DEBUG -O2 -mtune=cortex-a8 -march=armv7-a -c -o obj/PRU_PRUtoPRU_Interrupt.o PRU_PRUtoPRU_Interrupt.c 
PRU_PRUtoPRU_Interrupt.c: In function 'main':
PRU_PRUtoPRU_Interrupt.c:175:5: error: too few arguments to function 'prussdrv_pru_clear_event'
../../app_loader/include/prussdrv.h:182:9: note: declared here
PRU_PRUtoPRU_Interrupt.c:181:5: error: too few arguments to function 'prussdrv_pru_clear_event'
../../app_loader/include/prussdrv.h:182:9: note: declared here
make[1]: *** [obj/PRU_PRUtoPRU_Interrupt.o] Error 1
make[1]: Leaving directory `/root/pru/am335x_pru_package/pru_sw/example_apps/PRU_PRUtoPRU_Interrupt'
make: *** [all] Error 2
```

Tried on a BeagleBone (white) running kernel `3.8.13-bone35.2` and using Ubuntu FS. The whole process is described [here](https://github.com/erlerobot/wiki/wiki/Getting-started-with-the-PRU).

Has anybody experienced this before? is there a workaround?
Thanks

--- a/pru_sw/example_apps/PRU_PRUtoPRU_Interrupt/PRU_PRUtoPRU_Interrupt.c
+++ b/pru_sw/example_apps/PRU_PRUtoPRU_Interrupt/PRU_PRUtoPRU_Interrupt.c
@@ -172,13 +172,13 @@ int main(void)
     printf("\tINFO: Waiting for HALT command.\r\n");
     prussdrv_pru_wait_event (PRU_EVTOUT_0);
     printf("\tINFO: PRU0 completed transfer.\r\n");
-    prussdrv_pru_clear_event (PRU0_ARM_INTERRUPT);
+    prussdrv_pru_clear_event (PRU_EVTOUT_0, PRU0_ARM_INTERRUPT);
 
     /* Wait until PRU0 has finished execution */
     printf("\t\tINFO: Waiting for HALT command.\r\n");
     prussdrv_pru_wait_event (PRU_EVTOUT_1);
     printf("\t\tINFO: PRU1 completed transfer.\r\n");
-    prussdrv_pru_clear_event (PRU1_ARM_INTERRUPT);
+    prussdrv_pru_clear_event (PRU_EVTOUT_1, PRU1_ARM_INTERRUPT);
 
     /* Check if example passed */
     if ( LOCAL_examplePassed(PRU_NUM1) )

--- a/pru_sw/example_apps/PRU_memAccessPRUDataRam/PRU_memAccessPRUDataRam.c
+++ b/pru_sw/example_apps/PRU_memAccessPRUDataRam/PRU_memAccessPRUDataRam.c
@@ -145,7 +145,7 @@ int main (void)
     printf("\tINFO: Waiting for HALT command.\r\n");
     prussdrv_pru_wait_event (PRU_EVTOUT_0);
     printf("\tINFO: PRU completed transfer.\r\n");
-    prussdrv_pru_clear_event (PRU0_ARM_INTERRUPT);
+    prussdrv_pru_clear_event (PRU_EVTOUT_0, PRU0_ARM_INTERRUPT);
 
     /* Check if example passed */
     if ( LOCAL_examplePassed(PRU_NUM) )

--- a/pru_sw/example_apps/PRU_memAccess_DDR_PRUsharedRAM/PRU_memAccess_DDR_PRUsharedRAM.c
+++ b/pru_sw/example_apps/PRU_memAccess_DDR_PRUsharedRAM/PRU_memAccess_DDR_PRUsharedRAM.c
@@ -159,7 +159,7 @@ int main (void)
     printf("\tINFO: Waiting for HALT command.\r\n");
     prussdrv_pru_wait_event (PRU_EVTOUT_0);
     printf("\tINFO: PRU completed transfer.\r\n");
-    prussdrv_pru_clear_event (PRU0_ARM_INTERRUPT);
+    prussdrv_pru_clear_event (PRU_EVTOUT_0, PRU0_ARM_INTERRUPT);
 
     /* Check if example passed */
     if ( LOCAL_examplePassed(PRU_NUM) )

